docs/Regridding.rst

```diff
@@ -2,55 +2,126 @@
 Regridding
 **********
 
-Users that wish to interpolate their unipost output to a different grid
-may do so with the *wgrib2* utility. The general format for re-gridding
-to a lat-lon grid is given in the example.
+Users that wish to interpolate their unipost output to a different grid may do so with the *wgrib2*
+utility. The general format for re-gridding to various common projections are outlined in the following
+examples.
 
-==================
-Examples of wgrib2
-==================
-
-*Wgrib2* is a versatile program that has the ability to convert
-grib2 files from one grid to another for various user-defined grids as
-well as pre-defined NCEP grids. Complete documentation with examples of
-re-gridding for all available grid definitions can be found at:
+*Wgrib2* is a versatile program that has the ability to convert grib2 files from one grid to another
+for various user-defined grids as well as pre-defined NCEP grids. Complete documentation with examples
+of re-gridding for all available grid definitions can be found at:
 
 http://www.cpc.ncep.noaa.gov/products/wesley/wgrib2/new_grid.html.
```

Comment:
The period at the end of this link should be removed.

Reply:
Done, thanks